### DISCUSSION TITLE
How are translations supposed to be handled by the "consumer" of the questionnaire Bundle?

### DISCUSSION THREAD

**Discussion by fatso83** [helsenorge/structor#430] [Q&A] [answered]
I tried out the form designer functionality for translations and see that the exported result is a `Bundle` that consists of two `Questionnaire` resources. I do not see how there is structured data linking the two resources in any way, so I am wondering how you envision "consumers" of these bundles to choose between translations of a questionnaire? Right now, it looks like two separate questionnaires with no apparent commonality, apart from sharing some metadata (the `useContext` bit that seems to refer to the title of the "main" language's form).

I haven't looked at the source code of what is happening in the Preview mechanism, but it obviously must be doing something clever 😃 I would normally have thought that the translations would have been bundled along with a single Questionnaire in some manner.

Posted a [related question on the FHIR Community forum](http://community.fhir.org/t/supporting-multiple-languages-in-questionnaires/3862).

**Comment by kennethmyhra** [comment, marked as answer]
`Questionnaire.name` (which is a technical computer friendly name) is how we have decided to bind them together. We treat `Questionnaire.name` kind of like a unique form number/identity (blankettnummer). Other implementations have probably done it differently. I know at least the Finnish health portal was using a `uuid` that they put in the `Questionnaire.identifier`. And there are also completely different approaches to this. I think there is still some debate on how the standard wants to suggest translation support.

The language of the particular Questionnaire is put in `meta.tag` using a system of value: `urn:ietf:bcp:47`.

**Reply by kennethmyhra** [reply]
Updated the answer with details on where we identify the language of the Questionnaire
